### ISSUE TITLE
recovery fix

### ISSUE THREAD

**Issue by rdunski** [wmacevoy/brainwallet#8]
Found a bug in bw that gives syntax errors for recovering keys using the [2nd example](https://github.com/wmacevoy/brainwallet/blob/master/docs/usage.md) from documentation
```
bw --bits=160 --minimum=2 --shares=5 --key1="..." --key3="..." --secret --key2
```

**Comment by rdunski** [comment]
typo in brainwallet.py --> line 48: self.setPrime(***sefl***._millerRabin.prevPrime(2**bits)) Syntax
An easy fix, but the next one is a little trickier.
line 51: self._prime = Check.toPrime(value) --> NameError: global name 'value' is not defined

**Comment by lawalgren** [comment]
Fixed with commit 59f67a191a143747b9fb7e348c86877af20ceb0d
